### ISSUE TITLE
Suggestion: Post-upgrade Summary After Drupal 7 Migration

### ISSUE THREAD

**Issue by rbargerhuff** [backdrop/backdrop-issues#5606]
My colleague friend and I just encountered an inconvience that was due to the `project_installer` module not being enabled after updating the Drupal 7 database to Backdrop.

We strongly feel there should be some sort of post-update summary that lists modules that Backdrop CMS both supports and enables by default that were non-existent in Drupal 7. This sumnmary could include an enable button or a link to enable the particular module(s).

Cheers!

**Comment by argiepiano** [comment]
I think this is a good idea, @rbargerhuff. We recently merged a PR (available in the next release) that automatically enables contrib modules that are unique to Backdrop and required by other modules during upgrade - see #5499. But this stopped short of enabling B-only core modules, as there were some issues in doing so. However, a summary may be the way to go.  Any chance you may be able to tackle this?

**Comment by rbargerhuff** [comment]
> Any chance you may be able to tackle this?

Unfortunately not right now. I am currently leading 2 re-organization/migration projects that will take 2 of gigantic websites for the University I work for from Drupal 7 to Backdrop.

That being said, is there a list of B-only core modules like project_installer?

**Comment by argiepiano** [comment]
Take a look: https://github.com/backdrop/backdrop-issues/issues/5506

**Comment by rbargerhuff** [comment]
> Take a look: #5506

Thanks! I had to issue hop and found it. =) For us, after we migrated frm Drupal 7, we were missing `email`, `installer`, and `telemetry`.

**Comment by argiepiano** [comment]
Yup. Telemetry is not necessary - just send anonymous use data. Email: if you had the `email` contrib module enabled in D7, then this will be enabled too. And you know about `installer` already - that's the most serious one.

**Comment by argiepiano** [comment]
And funny - that issue was 5506, and this one is 5606  😄
